content/zh-cn/docs/images/tutor-service-nodePort-fig01.svg

```diff
@@ -0,0 +1 @@
ect height="24" width="35.078125"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml"><span style="" class="edgeLabel L-LS-node2' L-LE-node1" id="L-L-node2-node1">SNAT</span></div></foreignObject></g></g><g style="opacity: 1;" transform="translate(231.2109375,64)" class="edgeLabel"><g class="label" transform="translate(-17.5390625,-12)"><rect height="24" width="35.078125" ry="0" rx="0"></rect><foreignObject height="24" width="35.078125"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml"><span style="" class="edgeLabel L-LS-node1' L-LE-node2" id="L-L-node1-node2">SNAT</span></div></foreignObject></g></g><g style="opacity: 1;" transform="" class="edgeLabel"><g class="label" transform="translate(0,0)"><rect height="0" width="0" ry="0" rx="0"></rect><foreignObject height="0" width="0"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml"><span style="" class="edgeLabel L-LS-node1' L-LE-endpoint" id="L-L-node1-endpoint"></span></div></foreignObject></g></g></g><g class="nodes"><g style="opacity: 1;" transform="translate(38.5078125,42)" id="flowchart-client-10" class="node plain"><rect class="label-container" height="44" width="61.015625" y="-22" x="-30.5078125" ry="5" rx="5"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-20.5078125,-12)"><foreignObject height="24" width="41.015625"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">client</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(153.84375,42)" id="flowchart-node2-11" class="node k8s"><rect class="label-container" height="44" width="69.65625" y="-22" x="-34.828125" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-24.828125,-12)"><foreignObject height="24" width="49.65625"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">Node 2</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(308.578125,42)" id="flowchart-node1-15" class="node k8s"><rect class="label-container" height="44" width="69.65625" y="-22" x="-34.828125" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-24.828125,-12)"><foreignObject height="24" width="49.65625"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">Node 1</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(435.09375,42)" id="flowchart-endpoint-19" class="node k8s"><rect class="label-container" height="44" width="83.375" y="-22" x="-41.6875" ry="5" rx="5"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-31.6875,-12)"><foreignObject height="24" width="63.375"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">Endpoint</div></foreignObject></g></g></g></g></g></g></svg>
```

Comment:
This two svg file are not in use, maybe can be removed.

Reply:
just as tengqm said, we should reference copied file instead of "en/docs/image"